app.js

```diff
@@ -19,5 +21,9 @@ app.use((err, req, res, next) => {
 	}
 });
 
+app.use((err, req, res, next) => {
+	res.status(400).send({ msg: 'Bad request' });
```

Comment:
Instead of sending this response for all kinds of errors, please check the Psql error code and only send this when necessary.